File: ru.kmorozov.api.onedrive/src/main/java/com/wouterbreukink/onedrive/client/utils/JsonUtils.java
package com.wouterbreukink.onedrive.client.utils;

import com.google.api.client.json.JsonFactory;
import com.google.api.client.json.gson.GsonFactory;

/**
 * Created by sbt-morozov-kv on 13.03.2017.
 */
public class JsonUtils {

    public static final JsonFactory JSON_FACTORY = new GsonFactory();
}

File: ru.kmorozov.api.onedrive/src/main/java/ru/kmorozov/onedrive/client/utils/JsonUtils.kt
package ru.kmorozov.onedrive.client.utils

import com.google.api.client.json.JsonFactory
import com.google.api.client.json.gson.GsonFactory

/**
 * Created by sbt-morozov-kv on 13.03.2017.
 */
object JsonUtils {

    val JSON_FACTORY: JsonFactory = GsonFactory()
}
